nata/index.html

```diff
@@ -1,16 +1,50 @@
-<!doctype html>
-
+<!DOCTYPE html>
 <html lang="en">
-
-<head>
-  <meta charset="utf-8">
-  <title>Todo App</title>
-  <link rel="stylesheet" href="styles.css" />
-  <!-- If you uncomment the following line the design MUST be dark -->
-  <!-- link rel="stylesheet" href="styles.black.css" / -->
-</head>
-
-<body>
-
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="utf-8" />
+    <title>Todo App</title>
+    <link rel="stylesheet" href="styles.css" />
+    <!-- If you uncomment the following line the design MUST be dark -->
+    <!--link rel="stylesheet" href="styles.black.css" /-->
+  </head>
+  <body>
+    <header>
+      <h1>March 9, 2020</h1>
+      <p>5 incomplete, 5 completed</p>
+    </header>
+    <hr />
+    <main>
+      <section>
+        <h2>Incomplete</h2>
+        <input type="checkbox" />
+        <label class="incomplete">Upload 1099-R to TurboTax</label><br />
+        <p>💰 Finance</p>
+        <input type="checkbox" />
+        <label class="incomplete">Submit 2019 tax return</label><br />
+        <p>💰 Finance</p>
+        <input type="checkbox" />
+        <label class="incomplete">Print parking passes</label><br />
+        <p>💞 Wedding</p>
+        <input type="checkbox" />
+        <label class="incomplete">Sign contract, send back</label><br />
+        <p>🖥️ Freelance</p>
+        <input type="checkbox" />
+        <label class="incomplete">Hand sanitizer</label><br />
+        <p>🛒 Shopping List</p>
```

Comment:
I guest this must be a list. Every list item has `<input type="checkbox" />`, possible label and `<p>`